### ISSUE TITLE
Support downloading and bundling JDK

### ISSUE THREAD

**Issue by mrexodia** [TorgoTorgo/Ghidra-App-Bundle#9]
It looks like the launch script already supports an embedded JDK for the Graal thing. Would be cool if you could use `--embedjdk` to automatically download and extract the appropriate JDK from [here](https://jdk.java.net/18/)

**Comment by TorgoTorgo** [comment]
Yeah sure! I'll just query the architecture and grab the appropriate JDK. Thanks for the suggestion.
